lib/ai_helper/assistant.rb

```diff
@@ -312,18 +312,9 @@ def find_ai_helper_model(helper_mode, persona_klass)
 
       # Priorities are:
       #   1. Persona's default LLM
-      #   2. Hidden `ai_helper_model` setting, or `ai_helper_image_caption_model` for image_caption.
-      #   3. Newest LLM config
+      #   2. SiteSetting.ai_default_llm_id (or newest LLM if not set)
```

Comment:
```suggestion
      #   2. SiteSetting.ai_default_llm_id
```

I think we won't rely on this anymore thanks to ai_default_llm_model.